src/styles/text/text_labels.js

```diff
@@ -31,31 +31,58 @@ export const TextLabels = {
     parseTextFeature (feature, draw, context, tile) {
         // Compute label text
         let text = this.parseTextSource(feature, draw, context);
+
         if (text == null || text === '') {
             return; // no text for this feature
         }
 
         // Compute text style and layout settings for this feature label
         let text_settings = TextSettings.compute(feature, draw, context);
         let text_settings_key = TextSettings.key(text_settings);
-        let layout = this.computeTextLayout({}, feature, draw, context, tile, text, text_settings);
 
         // first label in tile, or with this style?
         this.texts[tile.key] = this.texts[tile.key] || {};
         let sizes = this.texts[tile.key][text_settings_key] = this.texts[tile.key][text_settings_key] || {};
 
-        // unique text strings, grouped by text drawing style
-        if (!sizes[text]) {
-            // first label with this text/style/tile combination, make a new label entry
-            sizes[text] = {
-                text_settings,
-                ref: 0 // # of times this text/style combo appears in tile
-            };
+        if (text instanceof Object){
```

Comment:
Related to earlier comment, I've been using `typeof` for checking `object`/`function`/`string`/`number`, but I don't know that I have a good reason for that vs. `instanceof` (for the object and function cases anyway... can't check string vs. number with `instanceof`). Do you know of any reasons to use one vs. the other? https://jbkflex.wordpress.com/2013/05/09/javascript-typeof-and-instanceof-the-concept-and-differences/